### DISCUSSION TITLE
Sample problem .e files

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#18551] [Q&A Getting Started] [answered]
I would like to run the example contact sliding block problem. Where do I find the exodus (sliding_elastic_blocks_2d.e) files needed to perform the analysis?

**Comment by GiudGiud** [comment, marked as answer]
You can find files with that name (which will contain the mesh) in the tests there:
/modules/contact/test/tests/frictional/sliding_elastic_blocks_2d/gold/sliding_elastic_blocks_2d_out.e
/modules/contact/test/tests/frictional/sliding_elastic_blocks_2d/sliding_elastic_blocks_2d.e
/modules/contact/test/tests/sliding_block/sliding/sliding_elastic_blocks_2d.e

Let me know if that s not the one you need

**Reply by TLWise** [reply]
Got it. I was able to find the files and run the code. Thank you.
